### DISCUSSION TITLE
qucik question:how to use some podman features

### DISCUSSION THREAD

**Discussion by devdailytester** [containers/podman#16909] [General]
i tried to use podman and i created a container.
however i dont know how to stop, it and it seems like a container created by toolbox cannot be stopped by podman?

Also how do I find out where is the container created stored?

I tried>
$ podman stop -a
07784000404bd3348ae4171f414c69710f3f470340d0a3c4f396b8268c1ee4a6
fa66d5e7fe95dd7c650a1be397964cdcb356bbd1a16aeeabc2d67542d047594b
[a@fedora Xyz]$ podman rmi 2110dbbc33d2
Error: Image used by 07784000404bd3348ae4171f414c69710f3f470340d0a3c4f396b8268c1ee4a6: image is in use by a container

extra log:
$ podman pod stop -a
$ podman rmi 2110dbbc33d2
Error: Image used by 07784000404bd3348ae4171f414c69710f3f470340d0a3c4f396b8268c1ee4a6: image is in use by a container
$ podman ps -ap
CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES POD ID PODNAME
07784000404b registry.fedoraproject.org/fedora-toolbox:36 toolbox --log-lev... 25 hours ago Created fedora-toolbox-36
fa66d5e7fe95 registry.fedoraproject.org/fedora:latest ls Less than a second ago Created busy_mcnulty

**Comment by rhatdan** [comment]
This looks like the remants of a buildah or some container in container-storage created by a tool outside of podman.

podman ps --all --external

Should show it.

$ podman rm --storage --all

Should remove it.

**Reply by rhatdan** [reply]
If you speficy the container name directly it will also be removed.  It is just not removed via --all, since this is removeing all `podman` containers.  Not buildah or other tools.

**Reply by RESDXChgfore9hing** [reply]
Thank you very much ,by name you mean something like :fedora-toolbox-36 ?

BTW, do you happen to know how can I get where is the path the container is stored.I wants to backup and move it to a external disk.
Is that possible?

**Comment by rhatdan** [comment]
podman rm 07784000404bd3348ae4171f414c69710f3f470340d0a3c4f396b8268c1ee4a6
Is what I am talking about.

**Reply by RESDXChgfore9hing** [reply]
ahhh i see thank you very much
